src/connector.rs

```diff
@@ -582,6 +582,12 @@ impl EthConnectorContract {
         sdk::return_output(0.to_string().as_bytes());
     }
 
+    /// Get accounts counter for statistics.
+    /// It represents total unique accounts (all-time, including accounts which now have zero balance).
+    pub fn get_accounts_counter(&self) {
+        sdk::return_output(self.ft.get_accounts_counter().to_string().as_bytes());
```

Comment:
Wouldn't this be more efficient to just have to `get_accounts_counter().to_le()`? Of course, the other end would need to read it as `le`. But, having them read it as a string feels odd to me.

Reply:
Done